assets/check

```diff
@@ -70,16 +70,12 @@ fi
 
 if [ -n "$tag_filter" ]; then
   {
-    if [ -n "$ref" ]; then
-      git tag --list "$tag_filter" --sort=creatordate --contains $ref
-    else
-      git tag --list "$tag_filter" --sort=creatordate | tail -1
-    fi
+    git tag --list "$tag_filter" --sort=creatordate | tail -1
```

Comment:
I'm worried about this change that gets rid of the looking for `$ref`. Can someone look at the implications of this?